.github/workflows/linux_conditional_compilation.yml

```diff
@@ -115,6 +115,7 @@ jobs:
       SELECTIVE_BUILD_STAT_DIR: /__w/openvino/openvino/selective_build_stat
       MODELS_PATH: /__w/openvino/openvino/testdata
       SCCACHE_AZURE_KEY_PREFIX: ubuntu22_x86_64_itt_clang_Release_faster_build
+    if: ${{ github.event_name != 'merge_group' }}
```

Comment:
It is a second `if`, there is one already above, this prevents the workflow from starting a pipeline: https://github.com/openvinotoolkit/openvino/actions/runs/14795450022.